plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportNativeAppEngineStandardProjectTest.java

```diff
@@ -66,8 +66,7 @@ public void importAppEngineStandardJava7_from1_3_1() throws IOException, CoreExc
     updateOldContainers();
 
     ProjectUtils.waitForProjects(project);
-    ProjectUtils.failIfBuildErrors("Imported App Engine standard Java 7 project has errors",
```

Comment:
I guess we could remove the call to `waitForProjects()`, couldn't we?

Reply:
`updateOldContainers()` is above, so I think it is safe to put it there. It may wait for some jobs to complete and there may be no errors.